controllers/servingruntime_controller.go

```diff
@@ -286,6 +301,50 @@ func (r *ServingRuntimeReconciler) Reconcile(ctx context.Context, req ctrl.Reque
 	return ctrl.Result{RequeueAfter: requeueDuration}, nil
 }
 
+func (r *ServingRuntimeReconciler) getPVCs(ctx context.Context, req ctrl.Request) ([]string, error) {
+	s := &corev1.Secret{}
+	if err := r.Client.Get(ctx, types.NamespacedName{
+		Name:      modelmesh.StorageSecretName,
+		Namespace: req.Namespace,
+	}, s); err != nil {
+		return nil, fmt.Errorf("Could not get the storage secret: %w", err)
+	}
+
+	pvcsMap := make(map[string]struct{})
+	var storageConfig map[string]string
+	for _, storageData := range s.Data {
+		if err := json.Unmarshal(storageData, &storageConfig); err != nil {
+			return nil, fmt.Errorf("Could not parse storage configuration json: %w", err)
+		}
+		if storageConfig["type"] == StoragePVCType {
+			pvcsMap[storageConfig["name"]] = struct{}{}
+		}
+	}
+	// add pvcs for predictors when the global config is enabled
+	if r.DeployPVCForPredictor {
```

Comment:
Per other comment this doesn't need to be a reconciler field, just pass `cfg` into this function and get the value from there.